socli/socli.py

```diff
@@ -361,7 +362,42 @@ def main():
         search.google_search = False
         tag = namespace.tag
         has_tags()  # Adds tags to StackOverflow url (when not using google search.
-
+    if namespace.open-url:
```

Comment:
I *think* this should be `namespace.open_url` (even though option name is `--open-url`) because `-` is a minus in python